src/mapnode.cpp

```diff
@@ -395,6 +395,77 @@ static std::vector<aabb3f> transformNodeBox(const MapNode &n,
 			boxes.push_back(box);
 		}
 	}
+	else if (nodebox.type == NODEBOX_CONNECTED)
+	{
+
```

Comment:
like:

```c++
size_t boxes_size = boxes.size();
boxes_size += box.size();
if (neighbors & 1) boxes_size += nodebox.connect_up.size();
/* repeat above line for all sides accordingly */
boxes.reserve(boxes_size);
```
of course with correct code style (if and the statement on separate lines)